.github/workflows/test-sanity.yml

```diff
@@ -78,16 +82,16 @@ jobs:
           rebase_or_merge: ${{ inputs.rebase_or_merge }}
           ps_dir: ${{ env.PS_DIR }}
 
-      - name: Pull mysql image in background
+      - name: Pull database image in background
         working-directory: ${{ env.PS_DIR }}
         run: |
-          # Pull mysql image
-          USER_ID=$(id -u) GROUP_ID=$(id -g) nohup docker compose -f docker-compose.yml pull -q mysql >& /dev/null &
+          # Pull ${{ inputs.db_type }} image
+          USER_ID=$(id -u) GROUP_ID=$(id -g) nohup docker compose -f docker-compose.yml pull -q ${{ inputs.db_type }}  >& /dev/null &
       - name: Build PrestaShop image in background
         working-directory: ${{ env.PS_DIR }}
         run: |
           # Build prestashop image in background
-          USER_ID=$(id -u) GROUP_ID=$(id -g) nohup docker compose -f docker-compose.yml build >& /dev/null &
+          USER_ID=$(id -u) GROUP_ID=$(id -g) nohup docker compose -f docker-compose.yml build prestashop-git >& /dev/null &
```

Comment:
Same question here, isn't there an env variable that is supposed to manage this correctly? Or is it that the default behaviour is to build all the defined services?
Maybe we can prevent this by adding `required: false` on both DB services